### PR TITLE
Mbst 16856 schedule query bug fix

### PR DESCRIPTION
**Pull request by jakekeeys**

### CODE REVIEW

File: src/test/java/org/atlasapi/persistence/content/schedule/mongo/MongoScheduleStoreTest.java [hunk 1]
```diff
@@ -249,11 +252,61 @@ private Item itemWithBroadcast(String id, Broadcast broadcast) {
...
         store.writeScheduleFrom(item2);
-        
+
         Schedule schedule = store.schedule(now.minusHours(4), now, ImmutableSet.of(BBC_ONE, BBC_TWO), ImmutableSet.of(Publisher.BBC, Publisher.C4, Publisher.ITV), Optional.<ApplicationConfiguration>absent());
         assertSchedule(schedule);
     }
-    
+
+    @Test
+    public void bothItemsShouldExistInTheSameScheduleBucket(){
```
Comment (MartinsIrbe) [inline src/test/java/org/atlasapi/persistence/content/schedule/mongo/MongoScheduleStoreTest.java]: Maybe add a test case where only one of the items should exist in the same schedule bucket? Are there some other possible test cases?

**Comment by emilssolmanis** [comment]
Commit messages could be better.

File: src/test/java/org/atlasapi/persistence/content/schedule/mongo/MongoScheduleStoreTest.java [hunk 2]
```diff
@@ -215,7 +217,8 @@ public void setUp() throws Exception {
     public void tearDown() {
         System.out.println("Completed in "+(System.currentTimeMillis()-when)+" millis");
     }
-    
+
```
Comment (MartinsIrbe) [inline src/test/java/org/atlasapi/persistence/content/schedule/mongo/MongoScheduleStoreTest.java]: remove whitespace

**Comment by emilssolmanis** [comment]
In fact, I'd say the commit messages need to be fixed before you merge this. The code itself lgtm, but we all need to get in the habit of providing context in the commits, e.g.,

https://github.com/atlasapi/atlas-persistence/pull/266
or
https://github.com/atlasapi/atlas-feeds/pull/224/commits/d1e599d0e99c0ee0ece6681be7b648d75eee1b6f

File: src/test/java/org/atlasapi/persistence/content/schedule/mongo/MongoScheduleStoreTest.java (hunk 1 above)
Comment (jakekeeys) [inline src/test/java/org/atlasapi/persistence/content/schedule/mongo/MongoScheduleStoreTest.java]: This is already covered by existing tests

**Pull request merged**